src/libraries/System.IO.Pipes.AccessControl/ref/System.IO.Pipes.AccessControl.cs

```diff
@@ -65,14 +65,4 @@ public partial class PipeSecurity : System.Security.AccessControl.NativeObjectSe
         public void SetAccessRule(System.IO.Pipes.PipeAccessRule rule) { }
         public void SetAuditRule(System.IO.Pipes.PipeAuditRule rule) { }
     }
-
-    public static class AnonymousPipeServerStreamAcl
```

Comment:
I now remember these APIs were added to .NET Core with the intention of bringing ACL support to pipes, which already existed in .NET Framework. 

There were other ACL cases in which I worked were the new extensions class needed to be consumable both from .NET Core and .NET Framework. It seems it's not the case with these two classes, so this change makes sense (within my limited understanding of the infra changes).